docs/sources/reference/run.md

```diff
@@ -426,36 +426,37 @@ the `--security-opt` flag. For example, you can specify the MCS/MLS level, a
 requirement for MLS systems. Specifying the level in the following command
 allows you to share the same content between containers.
 
-    # docker run --security-opt label:level:s0:c100,c200 -i -t fedora bash
+    $ sudo docker run --security-opt label:level:s0:c100,c200 -i -t fedora bash
```

Comment:
That's what @SvenDowideit suggested, atm we use `$ sudo docker run ...` for our examples, I haven't check other docs, maybe we can have a PR to change all of them.